app/core/api/urls.py

```diff
@@ -66,6 +67,11 @@
 router.register(
     r"user-status-types", UserStatusTypeViewSet, basename="user-status-type"
 )
+router.register(
+    r"project-stack-elements",
+    ProjectStackElementXrefViewSet,
+    basename="project-stack-elements",
```

Comment:
basename is used in url strings like <basename>-list and <basename>-detail. So it should be singular.
```suggestion
    basename="project-stack-element",
```